pandas/tests/arrays/test_datetimelike.py

```diff
@@ -1372,9 +1372,9 @@ def array_likes(request):
         data = memoryview(arr)
     elif name == "array":
         # stdlib array
-        from array import array
+        from array import array as array_stdlib
```

Comment:
@MarcoGorelli I wonder is there a way to make the pre-commit check not trigger in cases like this (seems like this might be a false positive)?

Reply:
yes...we can probably check that, if <name> has been imported from `pandas` _and_ there is `pd.`<name>, then trigger, but don't if it was imported from somewhere else

I do think I prefer aliasing `array` here if it's not from pandas, but I'll look into this nonetheless, thanks for bringing it up

Reply:
Personally I would say we should do ``import array`` / ``array.array(...)``. That should be clear?